src/plugins/intel_cpu/docs/internal_cpu_plugin_optimization.md

```diff
@@ -0,0 +1,75 @@
+# Internal CPU Plugin Optimizations
+
+The CPU plugin supports several graph optimization algorithms, such as fusing or removing layers.
+Refer to the sections below for details.
+
+> **NOTE**: For layer descriptions, see the [IR Notation Reference](https://docs.openvino.ai/latest/openvino_docs_ops_opset.html).
+
+
+## Fusing Convolution and Simple Layers
+
+Merge of a convolution layer and any of the simple layers listed below:
+- Activation: ReLU, ELU, Sigmoid, Clamp
+- Depthwise: ScaleShift, PReLU
+- FakeQuantize
+
+> **NOTE**: You can have any number and order of simple layers.
+
+A combination of a convolution layer and simple layers results in a single fused layer called 
+*Convolution*:
+
+![conv_simple_01](https://user-images.githubusercontent.com/26419192/159540184-62d3b3f6-74d8-4135-ba8e-c84da42922d3.png)
```

Comment:
Can we replace these external images on mermaid?

Reply:
I believe we can :) I guess for now we can include those external ones and take care of them post-merge.

Reply:
In general, now we can see all external dependencies in the one place, it is why I would like to replace all external images to mermaid. It will help to understand that we don't mis anything.

Reply:
I agree that we need to drop png files here, because they require extra LFS space and they totally not maintainable, while mermaid allows easily update the component architecture.

Reply:
Mermaid versions of these png files are in progress :)